.ocamlformat-ignore

```diff
@@ -0,0 +1,3 @@
+vendor/*/*
```

Comment:
Can you pull this change in a different PR?

Reply:
done in #1453